tdrs-backend/tdpservice/scheduling/db_backup.py

```diff
@@ -129,10 +145,12 @@ def upload_file(file_name, bucket, sys_values, object_name=None, region='us-gov-
     if object_name is None:
         object_name = os.path.basename(file_name)
 
+    logger.info(f"Uploading {file_name} to S3.")
     s3_client = boto3.client('s3', region_name=sys_values['S3_REGION'])
 
-    s3_client.upload_file(file_name, bucket, object_name)
-    print("Uploaded {} to S3:{}{}".format(file_name, bucket, object_name))
+    response = s3_client.upload_file(file_name, bucket, object_name)
+    logger.info(f"S3 upload response: {response}")
```

Comment:
@elipe17 should the response here be `None` if its successful? Asking because that's what I'm observing so far. 

```shell=
   2024-02-23T12:25:01.07-0500 [APP/PROC/WEB/0] ERR 2024-02-23 17:25:01,070 INFO db_backup.py::upload_file:L152 :  S3 upload response: None
   2024-02-23T12:25:01.07-0500 [APP/PROC/WEB/0] ERR [2024-02-23 17:25:01,070: INFO/ForkPoolWorker-1] S3 upload response: None
   2024-02-23T12:25:01.07-0500 [APP/PROC/WEB/0] ERR 2024-02-23 17:25:01,071 INFO db_backup.py::upload_file:L153 :  Uploaded /tmp/backup.pg to s3://...
```

Reply:
should return either True or False: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/s3-uploading-files.html. Would like to investigate more

Reply:
I looked at the [source code](https://boto3.amazonaws.com/v1/documentation/api/1.28.1/_modules/boto3/s3/transfer.html#S3Transfer.upload_file:~:text=def-,upload_file,-(%0A%20%20%20%20%20%20%20%20self%2C) for our version of the library and it looks like it doesnt return anything. That must be a feature added in a more recent version.